### PR TITLE
Work on Register functionality and user model

### PR DESCRIPTION
**Pull request by 2HEADSs**
Update user model accurate to the real idea.
Add functionality for register an user with token wich will be send back to the user. The changes are commented becasue will be used later. For now we use only hardcore date.

### CODE REVIEW

File: src/services/authService.js [hunk 1]
```diff
@@ -1,10 +1,91 @@
...
+        firstName: 'Michael',
+        lastName: 'Schumaher',
+        hashedPassword: await bcrypt.hash(password, 10)
+    });
+
     return user;
 };
 
+async function loginUser(email, password) {
+    const user = await User.findOne({ email });
```
Comment (Getsov) [inline src/services/authService.js:48]: check for isDeleted property

File: src/models/User.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 const { Schema, model } = require('mongoose');
-const emailValidator = /^[a-zA-Z0-9._%+-]{3,}@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
 
+
+//TODO: add validation for fields in the schema
 const userSchema = new Schema({
```
Comment (Getsov) [inline src/models/User.js:6]: is this the final user model ?

Comment (2HEADSs) [reply src/models/User.js:6]: You are the boss, I think it is enough for now!

Comment (Getsov) [reply src/models/User.js:6]: We are a team, I'm not a boss :)
I think "city" and "likedEvents" array/array of objects, which are optional, are necessary. Also, as we talked, "isDeleted" boolean property is mandatory with default status.

**Review by Getsov — APPROVED** [review]
Don't forget to comment out for TODO city and likedEvents for the user schema

**Pull request merged**